File: Models/Bookings.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace bookingsApp.Models
{
    public class Bookings
    {
        [Key]
        public int bookingID { get; set; }
        public DateTime createdOn { get; set; }
        public DateTime bookedFor { get; set; }
        public Boolean status { get; set; }
        [Timestamp]
        public byte[] RowVersion { get; set; }
        public Location location { get; set; }
        public User user { get; set; }

    }

}

File: ClientApp/src/app/Models/Bookings.ts
export interface Bookings{
    id: number,
    createdOn: string,
    bookedFor: string,
    bookedTime: string,
    status: string,
    revision: string,
    location: BookingLocation
};

export interface BookingLocation {
    id: number,
    name: string,
    phone: number
};